Generating new random routes...
Starting Webots on hostname
Warning: QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/local_scratch/pbs.3623189[1].pbs02/runtime-mwfranc'
Info: Sandboxing disabled by user.
AL lib: (WW) alc_initconfig: Failed to initialize backend "pulse"
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
INFO: sumo_supervisor: Starting controller: python3 -u sumo_supervisor.py --no-gui --no-netconvert --max-vehicles=100 --port=8873 --seed=0 --step=50 --radius=-1 --maximum-lateral-speed=2.5 --maximum-angular-speed=3 --lane-change-delay=3
INFO: av_m_controller: Starting controller: python3 -u av_m_controller.py
Connect to SUMO... This operation may take a few seconds.
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
Blind Factor: -0.8383634119503726   Distance :  1.2247125359826108  speed :  -0.006042519958083157  theta: -0.68453893245874
Blind Factor: -5.169405736072379   Distance :  9.265823652648487  speed :  -8.84943983852222  theta: -0.5579002935799223
Blind Factor: -0.7787314785831282   Distance :  1.0  speed :  0.4361679198828765  theta: -0.7787314785831282
Blind Factor: 2.251940434301492   Distance :  2.8139455846182386  speed :  6.818706174711317  theta: 0.8002786004858041
Blind Factor: 2.408029167867426   Distance :  3.0410478303062725  speed :  7.6851498395209195  theta: 0.7918419249673251
Blind Factor: -4.603077617615562   Distance :  9.569295488518959  speed :  -8.715542105191648  theta: -0.4810257581802379
Blind Factor: -0.794570501111214   Distance :  1.0  speed :  -0.16729625066995957  theta: -0.794570501111214
Blind Factor: 2.783686913656464   Distance :  3.568596958759622  speed :  7.87563209701026  theta: 0.7800508003078112
Blind Factor: -0.7920853089670483   Distance :  1.0  speed :  -0.17575363378947303  theta: -0.7920853089670483
Blind Factor: 2.5559751752812416   Distance :  3.1974264735764613  speed :  8.800948737373862  theta: 0.7993851293857186
Blind Factor: -1.037016091199841   Distance :  1.3169374730470278  speed :  0.27689729571804195  theta: -0.7874451995055419
Blind Factor: 2.8457289893970894   Distance :  3.644107887647112  speed :  9.813408405894844  theta: 0.7809123871012745
Blind Factor: -0.9981385818935755   Distance :  1.3169068273398736  speed :  -0.14672573017799753  theta: -0.7579416866641934
Blind Factor: 2.7339191275371646   Distance :  3.4825648923313817  speed :  10.623665572250884  theta: 0.7850303474767298
Blind Factor: -0.774537321569306   Distance :  1.0  speed :  0.06639017189829086  theta: -0.774537321569306
Blind Factor: 3.255996695777746   Distance :  4.077293214815543  speed :  10.020279254800457  theta: 0.7985681981238251
Blind Factor: -3.848472979064191   Distance :  8.675592668288  speed :  -8.608134431654914  theta: -0.44359770291331935
Blind Factor: -0.8343539708060463   Distance :  1.0806470376198953  speed :  -0.5812453375243843  theta: -0.7720874085248919
Blind Factor: 2.9164747321780764   Distance :  3.752486819906249  speed :  10.612979182829159  theta: 0.7772111861144234
Blind Factor: -0.9239080444537856   Distance :  1.203566415014947  speed :  -0.6306807424493497  theta: -0.7676419289601992
Blind Factor: -5.203535339446797   Distance :  9.182141525871566  speed :  -8.229717128735649  theta: -0.566701713841519
Blind Factor: -0.8940514524297037   Distance :  1.1313859168538902  speed :  -0.1204490122243843  theta: -0.7902267821362351
Blind Factor: -4.935630516996195   Distance :  8.879373714803577  speed :  -7.761385269261282  theta: -0.5558534504260785
Blind Factor: -0.7786619416997356   Distance :  1.0  speed :  -0.044066989104838486  theta: -0.7786619416997356
Blind Factor: -4.696501021891864   Distance :  8.803316301087156  speed :  -7.887226941639102  theta: -0.5334922501094133
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
ERROR: Cannot initialize the sound engine: Cannot initialize OpenAL default device 'OpenAL Soft'
Traceback (most recent call last):
  File "av_m_controller.py", line 527, in <module>
    risk_data_df.to_csv(proj_cwd + r"/output/data_"+timestamp+"_"+random_file_flag+
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 3386, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/formats/format.py", line 1083, in to_csv
    csv_formatter.save()
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/formats/csvs.py", line 228, in save
    with get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 642, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/home/mwfranc/Webots.HPC/Simulations/output/data_1632416255_818245_False.csv'
WARNING: 'av_m_controller' controller exited with status: 1.
Warning: The X11 connection broke (error 1). Did the X11 server die?
 
 
+------------------------------------------+ 
| PALMETTO CLUSTER PBS RESOURCES REQUESTED | 
+------------------------------------------+ 
 
mem=93gb,walltime=00:45:00,ncpus=5
 
 
+-------------------------------------+ 
| PALMETTO CLUSTER PBS RESOURCES USED | 
+-------------------------------------+ 
 
cput=02:14:55,mem=2750496kb,walltime=00:45:39,ncpus=5,cpupercent=284,vmem=21068008kb
 
 
